packages/app/playwright/lang.setup.ts

```diff
@@ -0,0 +1,15 @@
+import { test as setup } from '@playwright/test'
+
+import { BASE_URL } from '../playwright.config'
+
+setup('language settings', async ({ page }) => {
+  await page.goto(`${BASE_URL}  `)
+  await page.getByRole('button', { name: 'Home' }).click()
+  await page
+    .getByRole('button', { name: 'Admin User devnull@frachtwerk' })
```

Comment:
Do you think this could be error-prone because of `Admin User`? I can image that one could change the name for testing purposes and the test will fail 🤔 Another approach could be to introduce an E2E test user and work with him.

Reply:
Important question for all of our tests! With a dedicated E2E User we just shift the problem don't we? I think if we change the name for testing purposes we should also change it back, like in all E2E tests. So I would be fine with this till we might run into errors for any reason. :) 
What do you think?

Reply:
I think the admin user is more frequently in use for testing the UI etc. but a dedicated E2E just serves one purpose and is not supposed to be logged in with manually but only via an E2E test. So I think the probability of a failing E2E test based on the E2E user is much lower. What do you think @CathrinTruchan?

Reply:
Yes, I also think, that a dedicated user for E2E tests is a good idea!